packages/strapi-plugin-content-manager/controllers/ContentManager.js

```diff
@@ -139,8 +109,9 @@ module.exports = {
       state: { userAbility },
       params: { model, id },
     } = ctx;
+    const contentManagerService = strapi.plugins['content-manager'].services.contentmanager;
 
-    const { pm, entity } = await findEntityAndCheckPermissions(
+    const { pm, entity } = await contentManagerService.findEntityAndCheckPermissions(
```

Comment:
Please revert those changes and keep the findEntityAndCheckPermissions in this file as the CM refactor is changing already this code. We need to avoid conflicts as much as possible